File: C#/ByteBank/ByteBank.SistemaBanco.Core/SistemaInterno/ContaCorrente.cs
namespace ByteBank.SistemaInterno
{
    public class ContaCorrente : Conta
    {
        /// <summary>
        /// Construtor
        /// </summary>
        /// <param name="titular">Titular da <see cref="ContaCorrente"/>.</param>
        /// <param name="saldo">Saldo inicial da <see cref="ContaCorrente"/>.</param>
        public ContaCorrente(string titular, double saldo) : base(titular, saldo)
        {
            TipoConta = "Conta Corrente";
            Tarifa = 0.25;
            Saldo = saldo - (saldo * Tarifa);
        }
    }
}

File: Java/ByteBank/src/com/codealligator/interno/ContaCorrente.java
package com.codealligator.interno;

/**
 * Classe responsável pelo gerenciamento das contas correntes criadas.
 */
public class ContaCorrente extends Conta {

    /*
    Construtor.
     */
    public ContaCorrente(String titular, double saldo, double cpf, double numeroAgencia, double numeroConta) {
        super(titular, saldo, cpf, numeroAgencia, numeroConta);

        /*
        Seta a taxa/tarifa da Conta Corrente.
         */
        setTaxa(0.05);
    }
}
